io/src/lib.rs

```diff
@@ -4,32 +4,163 @@ use codec::{Decode, Encode};
 use gstd::{ActorId, String};
 use scale_info::TypeInfo;
 
-#[derive(Debug, Decode, Encode, TypeInfo)]
+#[derive(Debug, Decode, Encode, TypeInfo, Clone)]
 pub enum DaoAction {
+    /// Adds members to whitelist.
+    ///
+    /// Requirements:
+    /// * Only admin can add actors to whitelist;
+    /// * Member ID cant be zero;
+    /// * Member can not be added to whitelist more than once;
+    ///
+    /// Arguments:
+    /// * `member`: valid actor ID
+    ///
+    /// On success replies with [`DaoEvent::MemberAddedToWhitelist`]
     AddToWhiteList(ActorId),
```

Comment:
```suggestion
    /// On success replies with [`DaoEvent::MemberAddedToWhitelist`]
    AddToWhiteList(
        /// Valid actor ID
        ActorId
        ),
```
It's better to write descriptions for arguments exactly above arguments.